ethereum/p2p/src/main/java/tech/pegasys/pantheon/ethereum/p2p/netty/NettyP2PNetwork.java

```diff
@@ -125,6 +131,9 @@
   private static final Logger LOG = LogManager.getLogger();
   private static final int TIMEOUT_SECONDS = 30;
 
+  private final ScheduledExecutorService scheduledExecutorService =
```

Comment:
Should we give this a better name?  `peerDiscoveryExecutorService`?  `peerDiscoveryScheduler`?

Reply:
the name is rather uninspired :)